ogusa/parameter_plots.py

```diff
@@ -59,23 +61,24 @@ def plot_mort_rates(p, include_title=False, path=None):
     vals = ax.get_yticks()
     ax.set_yticklabels(['{:,.0%}'.format(x) for x in vals])
     if include_title:
-            plt.title('Mortality Rates')
+        plt.title('Mortality Rates')
     if path is None:
         return fig
     else:
         fig_path = os.path.join(path, "mortality_rates")
         plt.savefig(fig_path)
 
 
-def plot_pop_growth(p, start_year=2019, include_title=False,
-                    num_years_to_plot=150, path=None):
+def plot_pop_growth(p, start_year=2019, num_years_to_plot=150,
```

Comment:
@jdebacker . Why switch the order of these two arguments?

Reply:
I'm trying to keep the kwargs in a similar order across plotting functions and also group them in a way that makes sense (though I understand order doesn't matter when calling) - so I thought args dealing with the years plotted should be together and those with plot formatting/saving together.